packages/appstore/src/components/add-more-wallets/wallet-add-card.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { TWalletInfo } from 'Types';
+import { Text, WalletCard } from '@deriv/components';
+import { useCurrencyConfig } from '@deriv/hooks';
+import { observer, useStore } from '@deriv/stores';
+import { getWalletCurrencyIcon } from 'Constants/utils';
+import wallet_description_mapper from 'Constants/wallet_description_mapper';
+
+type TAddWalletCard = {
+    wallet_info: React.PropsWithChildren<TWalletInfo>;
+};
+
+const AddWalletCard = observer(({ wallet_info }: TAddWalletCard) => {
+    const {
+        ui: { is_dark_mode_on },
+    } = useStore();
+
+    const { currency, landing_company_name, is_added } = wallet_info;
+    const { getConfig } = useCurrencyConfig();
+    const currency_config = getConfig(currency);
+
+    const wallet_details = {
+        currency,
+        icon: currency_config?.is_crypto ? getWalletCurrencyIcon(currency, is_dark_mode_on) : currency_config?.icon,
```

Comment:
Why do we need to check for crypto here? I thought `getWalletCurrencyIcon` returns correct icons for crypto and fiat and you can use just this function. You can ignore it if I'm wrong

Reply:
`currency_config` doesn't return the icon I wanted for crypto. Instead I used to `getWalletCurrencyIcon` for cryto icons.